homeassistant/components/simplisafe/__init__.py

```diff
@@ -298,7 +298,7 @@ def _async_register_base_station(
     device_registry = dr.async_get(hass)
     device_registry.async_get_or_create(
         config_entry_id=entry.entry_id,
-        identifiers={(DOMAIN, system.system_id)},
+        identifiers={(DOMAIN, str(system.system_id))},
```

Comment:
Does this need a migration of existing device entries?

Reply:
#82993